numgo.go

```diff
@@ -11,8 +11,9 @@ import (
 
 // Array64 is an n-dimensional array of float64 data
 type Array64 struct {
-	shape        []uint64
-	strides      []uint64
+	shape        []int
+	strides      []int
+	offset       []int
```

Comment:
I think you removed strides accidentally instead of offset here.
I suppose strides stays, offset goes.